doc/deployment-process.md

```diff
@@ -25,14 +25,14 @@ The heading should link to a Github URL at the bottom of the file, which shows t
 1. Create a release branch and make a pull request
    - Create a branch from develop for the release called release-X where X is the release number
    - Update CHANGELOG.md to:
-     document the changes in this release in a bullet point form
-     add a link to the diff at the bottom of the file
-   - Document the changes in the commit message as well
+     - document the changes in this release in a bullet point form
+     - add a link to the diff at the bottom of the file
+   - Copy this release's changelog bullets into the commit message as well
```

Comment:
Suggest "changelog bullet points" instead of "changelog bullets", or better yet:
"Copy the list of changes from the changelog into the commit message as well"